internal/installation/context.go

```diff
@@ -0,0 +1,18 @@
+package installation
+
+import (
+	"github.com/ActiveState/cli/internal/errs"
+)
+
+type Context struct {
+	InstalledAsAdmin bool
+}
+
+func GetContext() (*Context, error) {
+	installedAsAdmin, err := getAdminInstallInformation()
```

Comment:
Nit: Why not implement this logic here? You already have platform specific files, just implement `GetContext()` specifically per platform. You're effecively already doing this through `getAdminInstallInformation()`.